### PR TITLE
infra: set PDB default value

### PR DESCRIPTION
**Pull request by busla**
- fix: added pdb default value
- fix: added pdb default value

# ...

Attach a link to issue if relevant

## What

Specify what you're trying to achieve

## Why

Specify why you need to achieve this

## Screenshots / Gifs

Attach Screenshots / Gifs to help reviewers understand the scope of the pull request

## Checklist:

- [ ] I have performed a self-review of my own code
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] Formatting passes locally with my changes
- [ ] I have rebased against main before asking for a review

### CODE REVIEW

File: infra/src/dsl/dsl.ts [hunk 1]
```diff
@@ -171,10 +172,7 @@ export class ServiceBuilder<ServiceType> {
    * @param pdb PodDisruptionBudget definitions
    */
   podDisruption(pdb: PodDisruptionBudget) {
-    this.serviceDef.podDisruptionBudget = {
-      ...this.serviceDef.podDisruptionBudget,
-      ...pdb,
-    }
+    this.serviceDef.podDisruptionBudget = pdb
     return this
```
Comment (AndesKrrrrrrrrrrr) [inline infra/src/dsl/dsl.ts:176]: You're removing the ability to override the PDB

File: infra/src/dsl/output-generators/map-to-helm-values.ts [hunk 2]
```diff
@@ -87,7 +87,7 @@ const serializeService: SerializeMethod<HelmService> = async (
   if (serviceDef.args) {
     result.args = serviceDef.args
   }
-  if (result.podDisruptionBudget) {
+  if (serviceDef.podDisruptionBudget) {
```
Comment (AndesKrrrrrrrrrrr) [inline infra/src/dsl/output-generators/map-to-helm-values.ts:90]: Good Bugfix :tada:

File: infra/src/dsl/output-generators/map-to-helm-values.ts [hunk 3]
```diff
@@ -63,7 +63,7 @@ const serializeService: SerializeMethod<HelmService> = async (
       )}`,
     },
     secrets: {},
-    podDisruptionBudget: serviceDef.podDisruptionBudget,
+    podDisruptionBudget: serviceDef.podDisruptionBudget ?? { minAvailable: 1 },
```
Comment (AndesKrrrrrrrrrrr) [inline infra/src/dsl/output-generators/map-to-helm-values.ts:66]: Sane defaults :heart:

Comment (AndesKrrrrrrrrrrr) [reply infra/src/dsl/output-generators/map-to-helm-values.ts:66]: Do we want this on _all_ envs?

Comment (busla) [reply infra/src/dsl/output-generators/map-to-helm-values.ts:66]: yes

**Pull request merged**